test/test_mpi.jl

```diff
@@ -0,0 +1,12 @@
+using MPI
+
+include("test_util.jl")
+
+@testset verbose=true "MPI helper functions" begin
+    @test TrixiBase.mpi_isparallel() == false
+    @test TrixiBase.mpi_isroot() == true
+
```

Comment:
```suggestion

    using MPI
    @test TrixiBase.mpi_isparallel() == false
    @test TrixiBase.mpi_isroot() == true
```
Would this make codecov happy?